src/Tribe/Main.php

```diff
@@ -172,8 +172,7 @@ public function load_assets() {
 				array( 'tribe-dependency', 'dependency.js', array( 'jquery', 'underscore' ) ),
 				array( 'tribe-dependency-style', 'dependency.css' ),
 				array( 'tribe-pue-notices', 'pue-notices.js', array( 'jquery' ) ),
-				array( 'tribe-jquery-ui-theme', 'vendor/jquery/ui.theme.css' ),
-				array( 'tribe-jquery-ui-datepicker', 'vendor/jquery/ui.datepicker.css' ),
+				array( 'tribe-datepicker', 'datepicker.css' ),
```

Comment:
Are those vendor libs required any place else? If they were only being used in relation to the previous datepicker, should we now purge them?